products/azure_rm_servicebusdisasterrecoveryconfig/api.yaml

```diff
@@ -0,0 +1,67 @@
+--- !ruby/object:Api::Product
+name: Azure Automation Variable
+prefix: arm
+versions:
+  - !ruby/object:Api::Product::Version
+    name: ga
+    base_url: NotUsedInAzure
+scopes:
+  - NotUsedInAzure
+azure_namespace: 'Microsoft.Automation'
+azure_version: 'stable/2015-10-31'
+objects:
+  - !ruby/object:Api::Resource
+    name: AutomationVariable
+    api_name: AutomationVariable
+    base_url: NotUsedInAzure
+
+    azure_sdk_definition: !ruby/object:Api::Azure::SDKDefinition
+      provider_name: Microsoft.Automation
+      create: !ruby/object:Api::Azure::SDKOperationDefinition
+        go_func_name: CreateOrUpdate
+        request:
+      read: !ruby/object:Api::Azure::SDKOperationDefinition
+        go_func_name: Get
+        request:
+        response:
+      update: !ruby/object:Api::Azure::SDKOperationDefinition
+        go_func_name: CreateOrUpdate
+        request:
+      delete: !ruby/object:Api::Azure::SDKOperationDefinition
+        go_func_name: Delete
+        request:
+
+    description: |
+      Manage Azure DisasterRecoveryConfig instance.
+    parameters:
+      - !ruby/object:Api::Azure::Type::str
+        name: 'resource_group_name'
+        description: 'Name of the Resource group within the Azure subscription.'
+        required: true
+        azure_sdk_references: ['']
+      - !ruby/object:Api::Azure::Type::str
+        name: 'namespace_name'
+        description: 'The namespace name'
+        required: true
+        azure_sdk_references: ['']
+      - !ruby/object:Api::Azure::Type::str
+        name: 'alias'
+        description: 'The Disaster Recovery configuration name'
+        required: true
+        azure_sdk_references: ['']
+      - !ruby/object:Api::Azure::Type::dict

```

Comment:
**minor** We can try to flatten the object here to make the interface clearer. I think we might need some additional inputs/metadata to define how to flatten the properties. For example, some properties should not be flattened (like `sku`) to construct a more logical structure.